octoprint_mrbeam/templates/laser_job_done.jinja2

```diff
@@ -6,15 +15,23 @@
             <h3 data-bind='visible: !is_dust_mode()'>{{ _('Laser Job Finished') }}</h3>
         </div>
         <div class="modal-body">
-            <img data-bind='visible: is_dust_mode()' src="/plugin/mrbeam/static/img/dusting.jpg">
-            <img data-bind='visible: !is_dust_mode()' src="/plugin/mrbeam/static/img/done.jpg">
-            <h5 data-bind='visible: is_dust_mode()'>Please wait a few seconds until Mr Beam II cleared most of the dust particles out of the working area.</h5>
-            <h5 data-bind='visible: !is_dust_mode()'>Done. Now check out what you have created with your Mr Beam II!</h5>
+            <!--Mr Beams map-->
+            <div id="mrbeams_map_container" class="z-depth-1-half map-container" style="height: 500px; width: 700px; display: none;">
```

Comment:
what happens if the according css is not loaded.

Reply:
I've put this in an if statement, so it will only be shown if DEV mode